src/styles/utils/_animations.scss

```
@@ -0,0 +1,11 @@
+@keyframes seconds {
+  to {
+    transform: rotate(450deg);
+  }
+}
+
+@keyframes minutes {
+  to {
+    transform: rotate(270deg);
+  }
+}
```

```suggestion
@keyframes rotate {
  from {
    transform: rotate(0)
  }
  
  to {
    transform: rotate(360deg)
  }
}
```
Make it work use transform-origin with correct value